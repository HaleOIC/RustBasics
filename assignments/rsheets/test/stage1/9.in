set A1 16 + 3 + 5
get A1
set A2 sleep_then(500, 15)
set A3 10
set A4 7
get A3
set B1 16
set B2 10
set B4 5
set A6 sum(A1_B4)
get A6