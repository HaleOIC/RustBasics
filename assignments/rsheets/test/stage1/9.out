A1 = 24
A3 = 10
A6 = 87
